api.swagger.yaml

```diff
@@ -3466,41 +3778,83 @@ paths:
-              schema:
-                type: string
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/successful-post-response-schema'
+        '200':
+          description: 'The idea information was updated successfully.'
         '400':
-          description: 'The list was not saved due to malformed JSON.'
+          description: 'The supplied UUID or JSON was malformed.'
         '403':
-          description: 'User is not authorized to create a list.'
+          description: 'User is not authorized to update this idea.'
+        '404':
+          description: 'The UUID does not correspond to an existing idea.'
         '422':
-          description: 'The list was not saved due to Drupal validation failure or JSON schema validation failure.'
+          description: 'The idea was not updated due to a Drupal validation failure or a JSON schema validation failure.'
       requestBody:
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/list-schema-create'
+              $ref: '#/components/schemas/idea-schema-update'
         required: true
       security:
         - SessionCookieHeader: []
         - APIKeyHeader: []
-        - CognitoJwt: [ invotra/lists:write ]
-  '/lists/list':
-    get:
+        - CognitoJwt: [ invotra/ideas:write ]
+    delete:
+      tags:
+        - 'ideas'
+      responses:
+        '200':
+          description: 'The idea was deleted successfully.'
+        '400':
+          description: 'The supplied UUID was malformed.'
+        '403':
+          description: 'The user does not have permission to delete the idea.'
+        '404':
+          description: 'The UUID does not correspond to an existing idea.'
+      description: Delete an idea.
+      security:
+        - SessionCookieHeader: []
+        - APIKeyHeader: []
+        - CognitoJwt: [ invotra/ideas:write ]
+  '/lists':
+    post:
+      tags:
+        - 'lists'
+      description: 'Provision a new list.'
+      responses:
+        '201':
+          description: 'The list was created successfully.'
+          headers:
+            Location:
+              description: 'The URL at which the newly provisioned list may be retrieved.'
+              schema:
+                type: string
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/successful-post-response-schema'
+        '400':
+          description: 'The list was not saved due to malformed JSON.'
+        '403':
+          description: 'User is not authorized to create a list.'
+        '422':
+          description: 'The list was not saved due to Drupal validation failure or JSON schema validation failure.'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/list-schema-create'
```

Comment:
I don't think 'uri' should be allowed when creating/updating lists. Just the UUID should be allowed. Happy for uri to be there on the GETs though

Reply:
done